packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -383,6 +384,8 @@ class _GlowController extends ChangeNotifier {
   @override
   void dispose() {
     _glowController.dispose();
+    _decelerator.removeListener(notifyListeners);
```

Comment:
Is it necessary to call `_decelerator.removeListener(notifyListeners);`?
Does not 'dispose' remove them?

If it is necessary, can you comment why?